etc/win-ci/crystal.iss

```diff
@@ -0,0 +1,335 @@
+Name: "samples"; Description: "Sample programs"; Types: full
+Name: "docs"; Description: "Offline standard library documentation"; Types: full
+
+[Tasks]
+Name: addtopath; Description: "Add Crystal's directory to the &PATH environment variable"; Flags: checkedonce
+Name: association; Description: "{cm:AssocFileExtension,Crystal,.cr}"; Flags: unchecked
+
+[Files]
+Source: "portable\{#MyAppExeName}"; DestDir: "{app}"; Flags: ignoreversion; Components: main
+Source: "portable\lib\*"; DestDir: "{app}\lib"; Flags: ignoreversion; Components: main
+Source: "portable\*.dll"; DestDir: "{app}"; Flags: ignoreversion; Components: main
+Source: "portable\src\*"; DestDir: "{app}\src"; Excludes: "llvm\ext\llvm_ext.obj"; Flags: ignoreversion recursesubdirs createallsubdirs; Components: main
+Source: "portable\README.md"; DestDir: "{app}"; Flags: ignoreversion isreadme; Components: main
+Source: "portable\LICENSE.txt"; DestDir: "{app}"; Flags: ignoreversion; Components: main
+
+Source: "portable\shards.exe"; DestDir: "{app}"; Flags: ignoreversion; Components: shards
+
+Source: "portable\crystal.pdb"; DestDir: "{app}"; Flags: ignoreversion; Components: pdb
+
+Source: "portable\samples\*"; DestDir: "{app}\samples"; Flags: ignoreversion recursesubdirs createallsubdirs; Components: samples
+
+Source: "portable\docs\*"; DestDir: "{app}\docs"; Flags: ignoreversion recursesubdirs createallsubdirs; Components: docs
+
+[UninstallDelete]
+Type: filesandordirs; Name: "{localappdata}\crystal\cache"
+
+[Registry]
+Root: HKA; Subkey: "Software\Classes\{#MyAppAssocExt}"; ValueType: string; ValueName: "PerceivedType"; ValueData: "text"; Flags: uninsdeletevalue
+Root: HKA; Subkey: "Software\Classes\{#MyAppAssocExt}\OpenWithProgids"; ValueType: string; ValueName: "{#MyAppAssocKey}"; ValueData: ""; Flags: uninsdeletevalue
+
+Root: HKA; Subkey: "Software\Classes\{#MyAppAssocKey}"; ValueType: string; ValueName: ""; ValueData: "{#MyAppAssocName}"; Flags: uninsdeletekey
+Root: HKA; Subkey: "Software\Classes\{#MyAppAssocKey}\DefaultIcon"; ValueType: string; ValueName: ""; ValueData: "{app}\{#MyAppExeName},0"; Flags: uninsdeletekey
+Root: HKA; Subkey: "Software\Classes\{#MyAppAssocKey}\shell\open\command"; ValueType: string; ValueName: ""; ValueData: """{app}\{#MyAppExeName}"" ""%1"""; Flags: uninsdeletekey
+
+Root: HKA; Subkey: "Software\Classes\Applications\{#MyAppExeName}\DefaultIcon"; ValueType: string; ValueName: ""; ValueData: "{app}\{#MyAppExeName},0"; Flags: uninsdeletekey
+Root: HKA; Subkey: "Software\Classes\Applications\{#MyAppExeName}\SupportedTypes"; ValueType: string; ValueName: ".cr"; ValueData: ""; Flags: uninsdeletekey
+
+[Icons]
+Name: "{group}\Crystal Standard Library API"; Filename: "{app}\docs\index.html"; Components: docs
+Name: "{group}\Crystal Standard Library API"; Filename: "https://crystal-lang.org/api/{#MyAppVersion}/index.html"; Components: not docs
+Name: "{group}\Official Website"; Filename: "https://crystal-lang.org/"

```

Comment:
Maybe a link to the Crystal book would actually be more useful than the website hompage? Or have both?